MatterControl.Printing/PrinterCommunication/PrinterConnection.cs

```diff
@@ -975,7 +972,8 @@ public void Connect()
 				return;
 			}
 
-			TerminalLog.Clear();
+			// TODO: Reimplement in application layer on connect
+			//TerminalLog.Clear();
```

Comment:
Create issue

Reply:
Issue created - MatterHackers/MCCentral#5767
Implement TerminalLog clear in application layer